hw/vendor/eslepfl_x_heep.vendor.hjson

```diff
@@ -8,7 +8,7 @@
 
   upstream: {
     url: "https://github.com/esl-epfl/x-heep.git",
-    rev: "27a283ccc00aed503a4b9d852caf42ca84dba562",
+    rev: "main",
```

Comment:
Always give a specific commit and not a branch name to make sure we can trace the specific version of the files that are currently working.